cdhweb/people/templates/people/snippets/profile_card.html

```diff
@@ -18,9 +18,10 @@
                     {% if grant %}
                     {# NOTE: for unknown reasons, on qa these are not returned as date objects; handle either date or YYYY-MM-DD string#}
                         <p>{% if profile.first_start and profile.last_end %}
+                        {# NOTE: not using start year because it creates incorrect display values; see issue 197 #}
                             {% firstof profile.first_start.year profile.first_start|slice:":4" as start_year %}
                             {% firstof profile.last_end.year profile.last_end|slice:":4" as end_year %}
-                            {{ start_year }}{% if start_year != end_year %}–{{ end_year }}{% endif %}
+                            {{ end_year }}
```

Comment:
seems a little weird to me not to display the start/end year range for the most recent grant, but I guess this works as a simple, temporary solution

Reply:
you're right! it also simplified things a ton to just use start/end for most recent grant. i've changed it. thanks!